app/models/chargeback_container_project.rb

```diff
@@ -62,7 +62,7 @@ def self.get_keys_and_extra_fields(perf, ts_key)
     project = @data_index.fetch_path(:container_project, :by_group_id, perf.resource_id)
     key = "#{project.id}_#{ts_key}"
 
-    [key, {"project_name"  => project.name}]
+    [key, {"project_name"  => project.old_ems_id.nil? ? project.name : "#{project.name} (Deleted)"}]
```

Comment:
> Yeah, definitely a fan of a model method of #archived?. And, that's probably a safer term than "deleted".

Ill go with an `archived?` method. Waiting for @dclarizio for `Archived` vs `Deleted`